core/src/main/kotlin/configuration.kt

```diff
@@ -14,6 +14,7 @@ object DokkaDefaults {
     val cacheRoot: File? = null
     const val offlineMode: Boolean = false
     const val failOnWarning: Boolean = false
+    const val globalMainPage: String = ""
```

Comment:
What should an empty String represent here? 
I would much rather go with something like optional file. If it is not specified, do not link anywhere 🤔